lib/ansible/modules/packaging/os/portage.py

```diff
@@ -490,9 +511,16 @@ def main():
             keepgoing=dict(default=False, type='bool'),
             jobs=dict(default=None, type='int'),
             loadavg=dict(default=None, type='float'),
+            quietbuild=dict(default=False, type='bool'),
+            quietfail=dict(default=False, type='bool'),
         ),
         required_one_of=[['package', 'sync', 'depclean']],
-        mutually_exclusive=[['nodeps', 'onlydeps'], ['quiet', 'verbose']],
+        mutually_exclusive=[
+            ['nodeps', 'onlydeps'],
+            ['quiet', 'verbose'],
+            ['quietbuild', 'verbose'],
+            ['quietfail', 'verbose']
```

Comment:
Could you please add trailing comma here?

Reply:
Sure I missed that, though I think there was not one before. I guess it needed it.

Reply:
It's just a thing, which makes diffs nicer when you add/remove list items. Otherwise the other line where you put comma is always listed as changed in diff, which makes it harder to review :)